libs/client/use-cases/part/list/shared/ui/src/lib/part-card/part-card.component.html

```diff
@@ -34,10 +34,17 @@ <h3 class="tui-island__paragraph bold">{{ partCard.name }}</h3>
       </p>
     </div>
     <div class="part-card__form card">
```

Comment:
![part-card](https://user-images.githubusercontent.com/23154054/215508039-8d6f91aa-6970-4b21-a6b8-0edd749bb6fd.PNG)

it seems to me that the button will look better if the price and availability section will be wrap with div with css properties: flex , justity-content: center, flex-direction: column